data/index.ts

```diff
@@ -30,7 +30,7 @@ export type QuantityType = {
   quantity_ingredient_name?: string;
   quantity_id: QuantityId;
   quantity_unit_id?: string;
-  quantity_unit: string;
+  quantity_unit?: string;
```

Comment:
```suggestion
  quantity_unit: string;
```

The `?` is to say it can be undefined. Sinc you defaultise it, it's not needed